lib/local-links-manager/import/import_comparer.rb

```diff
@@ -0,0 +1,32 @@
+class ImportComparer
```

Comment:
I like this new class 👍 We could probably add some unit tests for it? I guess it's already being indirectly tested in the Importer class tests, but would be nice to have explicit tests for its interface